packages/editor/src/components/editor-canvas/style.scss

```diff
@@ -0,0 +1,5 @@
+.edit-post-visual-editor {
```

Comment:
This class is edit-post specific, we should not use it here

Reply:
🤦

Reply:
Updated.